ecc_logic.py

```diff
@@ -388,6 +388,11 @@ def filter_morphs_below_threshold(morphlist, threshold):
 
 def get_all_morphnames_in_morphlists(morphlists):
     """ returns a list of alle morphnames found in the morphlists """
+

```

Comment:
This changes the order. Don't know if that is important. Also, I find it really hard to read and am not sure if it is valid code.